plugins/mac-avcapture/plugin-properties.m

```diff
@@ -135,7 +136,7 @@ bool properties_changed_use_preset(OBSAVCapture *capture, obs_properties_t *prop
         properties_changed_preset(capture, properties, preset_list, settings);
     }
 
-    const char *update_properties[5] = {"resolution", "frame_rate", "color_space", "video_range", "input_format"};
+    const char *update_properties[2] = {"frame_rate", "supported_format"};
```

Comment:
This is fine, but given that it's only 2 properties whose state needs to be changed it might be almost simpler to directly call `obs_property_set_<thing>` directly without the loop.